Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/ActionList.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DotNet.Maestro.WebApi.Models
+{
+    public class ActionsList
+    {
+        [JsonExtensionData]
+        private IDictionary<string, JToken> ActionNames { get; set; }
```

Comment:
I think that format is only used for fields, not properties.  In https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md, properties are named with Pascal case.